### PR TITLE
Had issues with XML comments causing issues.

### PR DESCRIPTION
**Pull request by AdamGrossTX**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md [hunk 1]
```diff
@@ -56,6 +56,9 @@ Deploy Removable Storage Access Control on Windows 10 devices that have antimalw
 
 You can use the following properties to create a removable storage group:
 
+> [!NOTE]
+> Comments using XML comment notation <!-- COMMENT --> can be used in the Rule and Group XML files, but they must be inside the first XML tag, not the first line of the XML file.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md]:
```suggestion
> Comments using XML comment notation /<!-- COMMENT --/> can be used in the Rule and Group XML files, but they must be inside the first XML tag, not the first line of the XML file.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @AdamGrossTX, kindly commit my suggested change. (Angle brackets must be escaped to appear in documentation. When they aren't, they may be interpreted as disallowed HTML.)

File: microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md [hunk 2]
```diff
@@ -56,6 +56,9 @@ Deploy Removable Storage Access Control on Windows 10 devices that have antimalw
 
 You can use the following properties to create a removable storage group:
 
+> [!NOTE]
+> Comments using XML comment notation /<!-- COMMENT --/> can be used in the Rule and Group XML files, but they must be inside the first XML tag, not the first line of the XML file.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md]:
```suggestion
> Comments using XML comment notation \<!-- COMMENT --\> can be used in the Rule and Group XML files, but they must be inside the first XML tag, not the first line of the XML file.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @AdamGrossTX, my bad. Could you please commit again? Many thanks.

**Review by denisebmsft — APPROVED** [review]
Thank you @AdamGrossTX

**Pull request merged**